epub4j-core/src/main/java/io/documentnode/epub4j/domain/LazyResource.java

```diff
@@ -1,7 +1,8 @@
 package io.documentnode.epub4j.domain;
 
+//import android.util.Log;
```

Comment:
Can we remove unused imports? If possible, it'd be great to not depend on Android libraries in this library.